ddtrace/contrib/internal/elasticsearch/patch.py

```diff
@@ -142,11 +142,6 @@ def _perform_request(func, instance, args, kwargs):
 
             span.set_tag(SPAN_MEASURED_KEY)
 
-            # Only instrument if trace is sampled or if we haven't tried to sample yet
-            if span.context.sampling_priority is not None and span.context.sampling_priority <= 0:
```

Comment:
We also have this logic in the new avro integration: https://github.com/DataDog/dd-trace-py/blob/v2.14.2/ddtrace/contrib/internal/avro/schema_iterator.py#L96 and protobuf integrations: https://github.com/DataDog/dd-trace-py/blob/v2.14.2/ddtrace/contrib/internal/protobuf/schema_iterator.py#L76. 

The algosearch: https://github.com/DataDog/dd-trace-py/blob/v2.14.2/ddtrace/contrib/internal/algoliasearch/patch.py#L133 and postgres integrations are also impacted: https://github.com/DataDog/dd-trace-py/blob/v2.14.2/ddtrace/contrib/internal/psycopg/extensions.py#L46

Reply:
Thanks! I'll create a follow-up PR for those other integrations